core/src/main/scala-2.13+/cats/data/NonEmptyLazyList.scala

```diff
@@ -47,7 +47,7 @@ object NonEmptyLazyList extends NonEmptyLazyListInstances {
     s.asInstanceOf[LazyList[A]]
 
   def fromLazyList[A](as: LazyList[A]): Option[NonEmptyLazyList[A]] =
-    if (as.nonEmpty) Option(create(as)) else None
+    if (as.nonEmpty) Some(create(as)) else None
 
   def fromLazyListUnsafe[A](ll: LazyList[A]): NonEmptyLazyList[A] =
```

Comment:
how do folks feel about this method throwing the exception potentially lazily (at first access, not at creation)? I'm not fond of the fact that this method gets rid of laziness, though I also understand that deferred exceptions thrown without context aren't super fun

Reply:
it's worth noting that in general, `LazyList` throws any exceptions later during access, not at creation

Reply:
@armanbilge thoughts?

Reply:
> how do folks feel about this method throwing the exception potentially lazily

Seems fine. It's an `unsafe` method, I think the bar is lower.